entitycore_svc/subnets.tf

```diff
@@ -0,0 +1,46 @@
+resource "aws_subnet" "entitycore_db_a" {
+  vpc_id            = var.vpc_id
+  availability_zone = "${var.aws_region}a"
+  cidr_block        = "10.0.17.0/28"
+  tags = {
+    Name = "entitycore_db_a"
+  }
+}
+
+resource "aws_subnet" "entitycore_db_b" {
+  vpc_id            = var.vpc_id
+  availability_zone = "${var.aws_region}b"
+  cidr_block        = "10.0.17.16/28"
+  tags = {
+    Name = "entitycore_db_b"
+  }
+}
+
+resource "aws_subnet" "entitycore_ecs_a" {
+  vpc_id            = var.vpc_id
+  availability_zone = "${var.aws_region}a"
+  cidr_block        = "10.0.17.32/27"
+  tags = {
+    Name = "entitycore_ecs_a"
+  }
+}
+
+resource "aws_subnet" "entitycore_ecs_b" {
+  vpc_id            = var.vpc_id
+  availability_zone = "${var.aws_region}b"
+  cidr_block        = "10.0.17.64/27"
+  tags = {
+    Name = "entitycore_ecs_b"
+  }
+}
+
+# give access to the internet so dockerhub can be reached
+resource "aws_route_table_association" "entitycore_ecs_a_docker_access" {
+  subnet_id      = aws_subnet.entitycore_ecs_a.id
+  route_table_id = var.internet_access_route_id
+}
```

Comment:
Maybe it's a good moment to start using AWS ECR instead of docker hub. If that is already the case - we can remove this code.